repose-aggregator/components/services/datastore-service/datastore-service-api/src/main/java/org/openrepose/core/services/datastore/DatastoreService.java

```diff
@@ -39,6 +40,7 @@
      * @return
      * @throws DatastoreUnavailableException if no datastore exists with the given datastoreName
      */
+    // TODO: Rename this to getDistributedDatastore for clarity?
     Datastore getDatastore(String datastoreName) throws DatastoreUnavailableException;
```

Comment:
theoritically this is going to strictly give you the dd, if you are in a non dd environment it should give you the local